workspace_hack/Cargo.toml

```diff
@@ -25,6 +25,7 @@ futures = { version = "0.3" }
 futures-channel = { version = "0.3", features = ["sink"] }
 futures-core = { version = "0.3" }
 futures-executor = { version = "0.3" }
+futures-io = { version = "0.3" }
```

Comment:
I don't see why this was added, am I missing something obvious?

Reply:
🤷 hakari added it at some point